chatformatter-core/src/main/java/com/eternalcode/formatter/config/PluginConfig.java

```diff
@@ -44,31 +49,43 @@ public class PluginConfig implements ChatSettings, PlaceholderStack, TemplateRep
         "# RGB and gradient message usage: ",
         "# <color:#ff00ee>Example message</color>",
         "# <gradient:#ff00ee:#f79459>Example message</gradient>",
-        " "
+        "# ",
+        "# You can use three more internal placeholders: <displayname> <name> <message>",
+        "# "
     })
-    public String defaultFormat = "{displayname} {arrow_right} {message}";
+    public String defaultFormat = "{displayname} » {message}";
+    @Description({ " ", "# Here you can set different formats for each rank.",
+            "# Remember! Rank name must be exact same like in you permission plugin configuration.",
```

Comment:
Rank name must be **exactly the same as in** your permission plugin configuration.